src/lints/pub_module_level_static_missing.ron

```diff
@@ -0,0 +1,47 @@
+SemverQuery(
+    id: "pub_module_level_static_missing",
+    human_readable_name: "pub module-level static is missing",
```

Comment:
Unlike consts, statics are not allowed to be associated with a type:
![image](https://github.com/obi1kenobi/cargo-semver-checks/assets/2348618/d771778e-29e3-4712-8d2c-aeab82b25ab9)

After looking into this, this limitation seems very unlikely to be changed anytime soon, since the semantics of such a feature would be quite a knot to untangle.

So in name of brevity let's remove `module level` from all the names and explanations, since all statics for the foreseeable future are always going to be module-level?